.github/workflows/docker.yml

```diff
@@ -0,0 +1,38 @@
+name: docker
```

Comment:
perhaps we could use a more telling name, like "build container image"?

Reply:
done!